treeherder/model/derived/jobs.py

```diff
@@ -1383,6 +1383,11 @@ def _load_ref_and_job_data_structs(
                 'option_collection_hash': option_collection_hash
             })
 
+        if created is True:
+            # A new ReferenceDataSignature has been added
+            for item in ExclusionProfile.objects.all():
+                item.save()
```

Comment:
I believe we need to update the lower_tier_signatures property (which is currently being passed as a parameter, after being generated earlier on). I think the best strategy here is to make `lower_tier_signatures` a member variable of the JobsModel class (instead of a function parameter), and add a member function to update it (using the existing code: https://github.com/mozilla/treeherder/blob/66ea865de690528166a8de0f43d88ebe69e86575/treeherder/model/derived/jobs.py#L993) as needed.

You'll also need to add a unit test for this, probably similar to your existing one (just testing that new jobs ingested have the new tier applied)